docs/pages/_app.tsx

```diff
@@ -28,7 +28,7 @@ export function reportWebVitals({
   label: string;
   value: number;
 }) {
```

Comment:
GitHub don't allow me to select the code block beyond the collapsible part (🤷‍♂️ ), so I cannot suggest a change properly, but it would be nice if you can replace the inline types definition for web metric with `NextWebVitalsMetric` type exported by Next:
* https://nextjs.org/docs/advanced-features/measuring-performance#typescript

Reply:
Nice! Thank you